engine/Shopware/Bundle/PluginInstallerBundle/Service/PluginLocalService.php

```diff
@@ -278,6 +278,10 @@ private function parseChangeLog($html)
     {
         $html = strip_tags($html, '<br><i><b><strong><em><del><u><div><span><ul><li><ll><ol><p><a>');
 
+        if (empty($html)) {
```

Comment:
```suggestion
        if ($html === '') {
```
we should for an empty string here. otherwise texts with `0` or `false` will also be skipped :wink:

Reply:
yeah good point.
`strip_tags` can only return a string and so only 0 is left and i think nobody will have a 0 standalone in there changelog, but this is cleaner.